src/ui/interactivity/gestures/drag-outside.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Drag outside an app
+description: How to drag from an app to another app or the operating system.
+---
+
+You will likely want to implement
+drag and drop somewhere in your app.
+
+You have a couple potential approaches
+that you can take. One directly uses
+Flutter widgets and the other uses a package
+available on pub.dev.
```

Comment:
**chore:** Use [semantic breaks](https://sembr.org/). These lines can go to 72 or 80 characters. These breaks are made a bit too short. Also, consider using a list here to make the passage easier to skim.

```suggestion
You have a couple potential approaches that you can take.

1. Use Flutter widgets.
1. Use a package available on pub.dev.
```